pkg/apis/serving/v1alpha1/revision_defaults_test.go

```diff
@@ -36,6 +37,33 @@ func TestRevisionDefaulting(t *testing.T) {
 				TimeoutSeconds:       defaultTimeoutSeconds,
 			},
 		},
+	}, {
+		name: "readonly volumes",
+		in: &Revision{
+			Spec: RevisionSpec{
+				Container: corev1.Container{
+					Image: "foo",
+					VolumeMounts: []corev1.VolumeMount{{
+						Name: "bar",
+					}},
+				},
+				ContainerConcurrency: 1,
+				TimeoutSeconds:       99,
+			},
+		},
+		want: &Revision{
+			Spec: RevisionSpec{
+				Container: corev1.Container{
+					Image: "foo",
+					VolumeMounts: []corev1.VolumeMount{{
+						Name:     "bar",
+						ReadOnly: true,
```

Comment:
Do we need a test, where RO: false and we check that it was overridden to true?

Reply:
Given the way Go defaults it feels like overkill.